common/js/survey.js

```diff
@@ -0,0 +1,36 @@
+
+$(function() {
+
+  // Fist, check if browser supports storing the data
+  if (typeof localStorage !== 'undefined') {
+
+    var surveyName = "Cloud survey 2 done";
+
+    // Only show if user didn't interact with the banner
+    if ( !localStorage.getItem(surveyName) ) {
+
+      // Build banner text
+      var $banner = $('<div class="bs-callout" style="display:none; text-align:left; max-width: 1024px; margin-top: 50px;"><h2>Hello Magento Community!</h2><p>We at Magento believe in listening to our community, as they play an important role in building valuable products in the future!  If you want to make an impact on Magento Cloud please share your feedback through this survey.</p><p>Thank you for advance participating in our survey. Your feedback is 100% confidential and will not be shared outside of Magento.</p> <button data-href="https://www.surveymonkey.com/r/CloudFeatures_LP2" class="btn btn-primary btn-large">OK, I will help</button> <button class="btn btn-large">No, thanks</button> </div>');
```

Comment:
```suggestion
      var $banner = $('<div class="bs-callout" style="display:none; text-align:left; max-width: 1024px; margin-top: 50px;"><h2>Hello Magento Community!</h2><p>We at Magento believe in listening to our community, as they play an important role in building valuable products in the future!  If you want to make an impact on Magento Cloud please share your feedback through this survey.</p><p>Thank you in advance for participating in our survey. Your feedback is 100% confidential and will not be shared outside of Magento.</p> <button data-href="https://www.surveymonkey.com/r/CloudFeatures_LP2" class="btn btn-primary btn-large">OK, I will help</button> <button class="btn btn-large">No, thanks</button> </div>');
```

Reply:
Thank you, i'll made the text of the banner little bit easier to read next time!